src/panels/config/repairs/ha-config-repairs-dashboard.ts

```diff
@@ -23,13 +39,54 @@ class HaConfigRepairsDashboard extends LitElement {
 
   @state() private _showIgnored = false;
 
+  @state() private _info?: SystemHealthInfo;
+
+  @state() private _resolutionInfo?: HassioResolution;
+
+  @state() private _supervisorStats?: HassioStats;
+
+  @state() private _coreStats?: HassioStats;
+
   private _getFilteredIssues = memoizeOne(
     (showIgnored: boolean, repairsIssues: RepairsIssue[]) =>
       showIgnored
         ? repairsIssues
         : repairsIssues.filter((issue) => !issue.ignored)
   );
 
+  public hassSubscribe(): Array<UnsubscribeFunc | Promise<UnsubscribeFunc>> {
```

Comment:
We always stay subscribed, but only send the info in the dialog, without updating the data in the dialog when it updates?

We should move the subscribe to the dialog.